src/Installers/Windows/WindowsHostingBundle/DotNetCore.wxs

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
     <Fragment>
+        <util:ProductSearch Id="DotNetRedistLtsProductSearch_x86"
+                            Condition="NOT VersionNT64"
+                            ProductCode="$(var.DotNetRedistLtsInstallerProductCodex86)"
+                            Result="version"
+                            Variable="DotNetRedistLtsProductVersion_x86" />
+
+        <util:ProductSearch Id="DotNetRedistLtsProductSearch_x64"
+                            Condition="VersionNT64"
+                            ProductCode="$(var.DotNetRedistLtsInstallerProductCodex64)"
+                            Result="version"
+                            Variable="DotNetRedistLtsProductVersion_x64" />
+
```

Comment:
How were things working without moving this `<Fragment/>` up❔ If everything was fine, let's limit the change to just the `UninstallCommand` additions.

Reply:
They weren't, the search wasn't firing. Both things were a bug

Reply:
Ah 🆗